doc/source/parameters/cfd/linear_solver_control.rst

```diff
@@ -0,0 +1,151 @@
+	  set amg preconditioner ilu relative tolerance = 1.00
+
+	  #---------------------------------------------------
+	  # other AMG solver parameters
+	  #---------------------------------------------------
+	  # AMG aggregation threshold
+	  set amg aggregation threshold = 1e-14
+
+	  # AMG number of cycles
+	  set amg n cycles              = 1
+
+	  # AMG w cycling. If this is set to true, W cycling is used. Otherwise, V cycling is used.
+	  set amg w cycles              = false
+
+	  # AMG smoother sweeps
+	  set amg smoother sweeps       = 2
+
+	  # amg smoother overlap
+	  set amg smoother overlap      = 1
+	end
+
+
+* The ``method`` parameter enables to choose an iterative solver for the linear system of equations. Lethe currently supports these core solution strategies:
+	* ``gmres`` (default parameter value), a GMRES iterative solver with ILU preconditioning.
+	* ``amg``, a GMRES iterative solver with AMG preconditioning and an ILU coarsener and smoother.
+	* ``bicgstab``, a BICGSTAB iterative solver with ILU preconditioning.
+	* ``direct``, a direct solver using `TrilinosWrappers::SolverDirect <https://www.dealii.org/current/doxygen/deal.II/classTrilinosWrappers_1_1SolverDirect.html>`_. 
+
+	.. tip:: 
+		On coarse meshes, the ``gmres``/``bicgstab`` solver with ILU preconditioning is more efficient. 
+
+		As the number of mesh elements increase, the ``amg`` solver is the most efficient. Generally, at 1M elements, the ``amg`` solver always outperforms the ``gmres`` or ``bicgstab`` for 2D steady-state problems.
+		
+		The use of ``direct`` solver should be avoided for 3D problems.
+
+* The ``verbosity`` option enables the display of the residual at each non-linear iteration, to monitor the progress of the linear iterations.
+
+.. note::
+	Example of a ``set verbosity = verbose`` output:
+
+	.. code-block:: text
+
+		-Tolerance of iterative solver is : 0.0429541
+		-Iterative solver took : 11 steps 
+		-Tolerance of iterative solver is : 3.62082e-05
+		-Iterative solver took : 16 steps 
+		-Tolerance of iterative solver is : 1.05775e-08
+		-Iterative solver took : 17 steps 
+		-Tolerance of iterative solver is : 1.00205e-12
+		-Iterative solver took : 16 steps 
+		-Tolerance of iterative solver is : 1e-13
+		-Iterative solver took : 5 steps 
+		-Tolerance of iterative solver is : 1e-13
+		-Iterative solver took : 0 steps 
+
+
+* The ``minimum residual`` for the linear solver.
+
+* The ``relative residual`` for the linear solver.
+.. tip::
+	A good rule of thumb is to set the linear solver ``minimum residual`` at least :math:`10` times (preferably :math:`100` times) smaller than the `Non-linear solver <https://lethe-cfd.github.io/lethe/parameters/cfd/non-linear_solver_control.html>`_ ``tolerance`` parameter, and keep the relative residual reasonable, for instance ``set relative residual = 1e-3``. To lower the computational cost for more complex simulations, it can be lowered to ``set relative residual = 1e-4``.
```

Comment:
```suggestion
	A good rule of thumb is to set the linear solver ``minimum residual`` at least :math:`10` times (preferably :math:`100` times) smaller than the `Non-linear solver <https://lethe-cfd.github.io/lethe/parameters/cfd/non-linear_solver_control.html>`_ ``tolerance`` parameter, and keep the relative residual reasonable, for instance ``set relative residual = 1e-3``. To lower the computational cost for more complex simulations, it can be lowered to ``set relative residual = 1e-4``.
```

Can you use the :doc: command to refer to the documentation instead of an URL? This way this works on a local docuementation.

Reply:
sure! I did not know this command, thank you for the suggestion, this will be cleaner indeed ;)